backend/app/assets/stylesheets/spree/backend/components/_breadcrumb.scss

```diff
@@ -1,4 +1,16 @@
 .breadcrumb {
   font-size: 16px;
   margin-bottom: 0;
+  
+  a {
+    color: $color-8;
```

Comment:
Please add a variable for this use case and give it a default of `$color-8` instead of using the color variable directly.